hs_core/views/discovery_json_view.py

```diff
@@ -0,0 +1,59 @@
+import simplejson as json
```

Comment:
why are you not using the json library that already is packaged with Python in its standard library?